azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -718,20 +748,26 @@ def get_twin(self, device_id):
         """
         return self.protocol.devices.get_twin(device_id)
 
-    def replace_twin(self, device_id, device_twin):
+    def replace_twin(self, device_id, device_twin, etag=None):
```

Comment:
The API supports etags, but we did not expose it to the user before for some reason.